### ISSUE TITLE
login screen loads on 2ndry monitor

### ISSUE THREAD

**Issue by xGoatAlpha** [elementary/greeter#231]
The login screen in juno is now located on the secondary monitor (dual monitor setup). The os loads fine on the correct monitor after login. This wouldn't be such a problem if it weren't for the fact that you now also need to click the login password box before typing the password too making login impossible without powering on the 2nd monitor.

**Comment by xomp** [comment]
I too have this issue as well as it's doing some silly zooming/scaling with my desktop wallpaper during the sign in process. Once the sign in process is complete, the wallpaper is fine.

**Comment by xomp** [comment]
I managed to fix this by doing the following:

1. Find out your primary display using xrandr in terminal. Take note of it's name.
2. Create a file named 'correct-screen-pantheon-greeter' in /usr/local/bin/ and chmod +x that file.
3. Enter the below content into 'correct-screen-pantheon-greeter':
`#!/bin/bash
#
# { 2017-07-02 16:20 }
# { Ryan Miller | ryan@abstractfoc.us }
# { /usr/local/bin/correct-screen-pantheon-greeter }
# 
# Set correct primary monitor for login screen when lightdm greeter starts up on Elementary OS.

# Change this variable to your primary display (Run 'xrandr -q' to see available displays)
PRIMARY_DISPLAY='HDMI-0'

# Get list of connected monitors
x_out=$(xrandr -q | grep -e '\( \)connected' | awk -F' ' '{ print $1 }')

# Safety - Check if PRIMARY_DISPLAY is connected; if not, bail out of script
[[ $(grep ${PRIMARY_DISPLAY} <<< ${x_out}) ]] || exit 0
     
# Iterate over x_out and disable all monitors except for PRIMARY_DISPLAY
for monitor in ${x_out[@]} ; do
    if [[ ${monitor} = ${PRIMARY_DISPLAY} ]] ; then
        xrandr --output ${monitor} --primary
    else
        xrandr --output ${monitor} --off
    fi
done`

4. Create a file named 'lightdm.conf' in /etc/lightdm/ and chmod +x it. Now put the below content in this new file:
`#{ /etc/lightdm/lightdm.conf }
[Seat:*]
greeter-setup-script=/usr/local/bin/correct-screen-pantheon-greeter`

That's it! Works a treat. Not really sure why someone from the eOS team can't implement this fix so the users aren't having to be bothered with it.

**Comment by vytenis-nfq** [comment]
@xomp is it possible to you create a pull request to fix that issue? :zzz:

**Comment by peteruithoven** [comment]
Seems like a duplicate of #65. 
Note that the password focus issue has been fixed.

**Comment by FDiskas** [comment]
Still reprodused
